src/Simulation/TargetDefinitions/TargetPackages/Common.Package.props

```diff
@@ -23,7 +22,7 @@
   <ItemGroup>
     <ProjectReference Include="..\TargetDefinitions\Interfaces\Microsoft.Quantum.Targets.Interfaces.csproj" />
     <ProjectReference Include="..\QSharpFoundation\Microsoft.Quantum.QSharp.Foundation.csproj" />
-    <ProjectReference Include="..\CSharpGeneration\Microsoft.Quantum.CSharpGeneration.fsproj" PrivateAssets="All" IsQscReference="true" />
+    <ProjectReference Include="..\Core\Microsoft.Quantum.Runtime.Core.csproj" />
```

Comment:
How come the extra reference to Core is needed in some of these project files?

Reply:
They need that to resolve some of the types defined in the core, but it's not clear to me how they were getting it before, since CSharpGeneration doesn't include that package. After making the other updates I'll see if this is still needed...

Reply:
The errors end up looking like this:
```
obj/qsharp/__AutoSubstitution__.g.cs(33,49): error CS0012: The type 'Qubit' is defined in an assembly that is not referenced. You must add a reference to assembly 'Microsoft.Quantum.Runtime.Core, Version=0.16.2106.2312, Culture=neutral, PublicKeyToken=null'. [/home/vsts/work/1/s/src/Simulation/AutoSubstitution.Integration.Tests/Tests.Microsoft.Quantum.AutoSubstitution.Integration.csproj]
```

Reply:
After shuffling some things around, the explicit reference to the core is no longer needed, and this should now compile without it.